packages/x-date-pickers/src/DateCalendar/index.ts

```diff
@@ -4,6 +4,7 @@ export type {
   DateCalendarSlots,
   DateCalendarSlotProps,
 } from './DateCalendar.types';
+export type { PickerDayOwnerState as PickersDayOwnerState } from './DayCalendar';
```

Comment:
Do we need this alias? 🤔 
```suggestion
export type { PickerDayOwnerState } from './DayCalendar';
```

Reply:
Nop, it's a mistake while removing the plural form, I'll remove it